src/toil/test/sort/restart_sort.py

```diff
@@ -0,0 +1,263 @@
+# Copyright (C) 2015-2016 Regents of the University of California
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A demonstration of toil. Sorts the lines of a file into ascending order by doing a parallel merge sort.
```

Comment:
This ought to explain that this is an intentionally buggy version and not a complete duplicate.